build/import/VisualStudio.props

```diff
@@ -25,6 +25,7 @@
     <PackageReference Include="Microsoft.VisualStudio.RpcContracts" />
     <PackageReference Include="Microsoft.VisualStudio.Threading.Analyzers" />    
     <PackageReference Include="Microsoft.VisualStudio.Shell.15.0" />
+    <PackageReference Include="Microsoft.VisualStudio.Shell.Design" />
```

Comment:
This reference was now needed in the Microsoft.VisualStudio.ProjectSystem.Managed.VS project, not just Microsoft.VisualStudio.Editors and Microsoft.VisualStudio.AppDesigner.